fastlane/lib/fastlane/actions/spaceship_logs.rb

```diff
@@ -0,0 +1,132 @@
+module Fastlane
+  module Actions
+    class SpaceshipLogsAction < Action
+      def self.run(params)
+        latest = params[:latest]
+        print_contents = params[:print_contents]
+        print_paths = params[:print_paths]
+        copy_to_path = params[:copy_to_path]
+        copy_to_clipboard = params[:copy_to_clipboard]
+
+        # Get log files
+        files = Dir.glob("/tmp/spaceship*.log").sort_by { |f| File.mtime(f) }.reverse
```

Comment:
Can we get the log path from spaceship itself somehow so this doesn't "recreate" the log file save location here?

Reply:
I looked and its a constant anywhere but I could extract that out 💪

Reply:
What if there's no spaceship logs at all? 🤔